packages/fxa-settings/src/components/UnitRow/index.test.tsx

```diff
@@ -83,4 +83,28 @@ describe('UnitRow', () => {
       'Create'
     );
   });
+
+  it('renders `secondaryCtaText`', () => {
+    renderWithRouter(
+      <UnitRow
+        header="Display name"
+        headerValue={null}
+        noHeaderValueText="Not set"
+        ctaText="Create"
+        secondaryCtaText="Disable"
```

Comment:
I think you can take `secondaryCtaText="Disable"` out since it gets "Disable" by default.

I also think `ctaText` can be removed, it should get "Create" when `headerValue` is falsey.